docs/loading/Spark-connector-starrocks.md

```diff
@@ -2,12 +2,22 @@
 
 StarRocks provides a self-developed connector named StarRocks Connector for Apache Spark™ (Spark connector for short) to help you load data into a StarRocks table by using Spark. The basic principle is to accumulate the data and then load it all at a time into StarRocks through [STREAM LOAD](../sql-reference/sql-statements/data-manipulation/STREAM%20LOAD.md). The Spark connector is implemented based on Spark DataSource V2. A DataSource can be created by using Spark DataFrames or Spark SQL. And both batch and structured streaming modes are supported.
 
+> **NOTICE**
+>
+> Only users with the SELECT and INSERT privileges on a StarRocks table can load data into this table. You can follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the required privileges.
+
 ## Version requirements
 
 | Spark connector | Spark            | StarRocks     | Java | Scala |
 | --------------- | ---------------- | ------------- | ---- | ----- |
+| 1.1.1 | 3.2, 3.3, or 3.4 | 2.5 and later | 8 | 2.12 |
 | 1.1.0           | 3.2, 3.3, or 3.4 | 2.5 and later | 8    | 2.12  |
 
+> **NOTICE**
+>
+> - Please see [Upgrade Spark connector](#upgrade-spark-connector) for behaviour changes among different versions of the Spark connector.
+> - The Spark connector does not provide MySQL JDBC driver since version 1.1.1, and you need import the driver to the spark classpath manually. You can find the driver on [MySQL site](https://dev.mysql.com/downloads/connector/j/) or [Maven Central](https://repo1.maven.org/maven2/mysql/mysql-connector-java/).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/d3845eeb-c514-40c1-b4dc-8f1eb74572fa)
-> need to import